src/store/app/app.test.ts

```diff
@@ -66,4 +75,24 @@ describe('app store', () => {
       expect(currentBlockSelector(store.getState())).toBe(undefined);
     });
   });
+
+  describe('chainsInCurrentNetwork', () => {
```

Comment:
maybe add test for `chainId === undefined` and is there any option that chainId isn't in both?

Reply:
no, there is no way that one chainId will exist in both